.github/workflows/bin-image.yml

```diff
@@ -104,7 +104,7 @@ jobs:
         uses: docker/setup-buildx-action@v2
       -
         name: Login to Docker Hub
-        if: github.event_name != 'pull_request'
+        if: github.event_name != 'pull_request' && github.repository == 'moby/moby'
```

Comment:
This makes me curious if actions have a property to indicate the _type_ of repository it's running on. GitHub already has a concept of "source" repositories (i.e., the "original"), "forked" repositories, and various other ones.

If that's available, these could use such properties, instead of having to hard-code a repository-name 🤔 

<img width="365" alt="Screenshot 2023-08-19 at 11 16 21" src="https://github.com/moby/moby/assets/1804568/9763d301-6f66-4cc7-aa0e-70820dfa2f2f">

Reply:
The [docs](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context) aren't [easy](https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads#push) to [follow](https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#get-a-repository), but perhaps `github.event.repository.fork` will work?